observelib/observe/src/main/java/org/ballerinalang/observe/nativeimpl/OpenTracerBallerinaWrapper.java

```diff
@@ -36,22 +36,22 @@
 import java.util.concurrent.atomic.AtomicLong;
 
 import static io.ballerina.runtime.observability.ObservabilityConstants.DEFAULT_SERVICE_NAME;
+import static io.ballerina.runtime.observability.ObservabilityConstants.TAG_KEY_ENTRYPOINT_FUNCTION_MODULE;
+import static io.ballerina.runtime.observability.ObservabilityConstants.TAG_KEY_ENTRYPOINT_FUNCTION_POSITION;
 
 /**
  * This class wraps opentracing apis and exposes extern functions to use within ballerina.
  */
 public class OpenTracerBallerinaWrapper {
+    static final int ROOT_SPAN_INDICATOR = -2;
     private static final Logger log = LoggerFactory.getLogger(OpenTracerBallerinaWrapper.class);
-
     private static final OpenTracerBallerinaWrapper instance = new OpenTracerBallerinaWrapper();
+    private static final int SYSTEM_TRACE_INDICATOR = -1;
     private final TracersStore tracerStore;
     private final boolean enabled;
     private final Map<Long, ObserverContext> observerContextMap = new HashMap<>();
     private final AtomicLong spanIdCounter = new AtomicLong();
 
-    private static final int SYSTEM_TRACE_INDICATOR = -1;
-    static final int ROOT_SPAN_INDICATOR = -2;
-
```

Comment:
Any reason for this change ? Shall we revert this ?
I think the before the constants were kept separately.

Reply:
It was due to auto-formatting. Will revert this change.

Reply:
FIxed with 3a5fad4b8ee95545ae441daea7a28e24052367f6